molecule/os_hardening_vm/molecule.yml

```diff
@@ -8,7 +8,7 @@ driver:
   provider:
     name: libvirt
 platforms:
-  - name: instance
+  - name: "${USER}"
```

Comment:
Explaination:
When running in different runner instances on same machine, we need different names for the started VMs. Before we had a universal name `instance`. My approach was to name the VM the same as the tested OS. This created problems, since the two runner users would pick up different OSes random and one user cannot destroy another users VM. 

Letting everything run under the same user in parallel is not really well supported by molecule. so we need two different users.

By using the username as name for the VM we can ensure, that one runner alwys uses the same name for their test VM. THis wayeverything works out, tests can run in parallel and workflows can be aborted and restarted.

Reply:
Please add the explanation as a comment in the code. :)